### PR TITLE
All files are added to Portfolio branch

### PR DESCRIPTION
**Pull request by Alynn9**

### CODE REVIEW

File: Home.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        <meta name="viewport" content="width= device-width, initial-scale=1.0">
+        <title>Alynn Torossian</title>
+        <link rel="icon" type="image/x-icon" href="/mf-logo2 white.svg"/>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="Home.css">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Russo+One&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <div class="navBar">
```
Comment (Chrishalaby) [inline Home.html:18]: use sections instead of div tags

File: Home.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                    <img src= "/mf-logo2.svg" alt="logo">
+                </div>
+                <div class="navBar navbar__text">
+                    <li class="navBar navbar--text1"><a href="#">Mentorship</a> </li>
+                    <li class="navBar navbar--text2"><a href="#">Say Hello</a> </li>
+                </div>
+            </ul>
+        </div>
+        <Section class="text-is-center">
+            <div class="text-is-center container">
```
Comment (Chrishalaby) [inline Home.html:30]: no need for the concatenated divs.

File: Home.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+                <div class= "text-is-center title-sub">
+                    <h1>Designer,Front-end developer & Student</h1>
+                    <p>I design and code beautifully simple things, and i love what i do </p>
+                    <img class="text-is-centerme" src="/my logo.svg">
+                </div>
+            </div>
+            <div class="electronics">
+                <div class="container">
```
Comment (Chrishalaby) [inline Home.html:38]: same as before, check the rest of the code for unneeded concatenation.

File: Home.html [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+                    quietly confident, naturally curious, and perpetually working on improving my chops one design problem at a time.</p>
+                </div>
+            </div>
+        </section>
+
+        <section class="big__box">
+            <div class="box">
+                <div class="box1">
+                    <img class="text-is-centerme --modify1" src="/designer.svg" alt="designer">
+                    <h1>Designer</h1>
```
Comment (Chrishalaby) [inline Home.html:58]: h1 tag should only be used once and followed by the rest of the headers from h2 to h5

File: Home.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+                        <p>I value simple content structure, clean design patterns, and thoughtful interactions.</p>
+                    </div>
+                    <div class="box__text --modify2">
+                        <p>Things I enjoy designing:</p>
+                    </div>        
+                    <div class="box__text">
+                        <p>Web and Logos</p>
+                    </div>     
+                </div>
+                <div class="box2">
```
Comment (Chrishalaby) [inline Home.html:69]: way too many divs that can be replaced with a ul tag

File: Home.html [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+            <div class="content2">
+                <p>Interested in working together? We should queue up a chat. I’ll buy the coffee.</p>
+            </div>
+            <div class="content3">
+                <h3>Let's do this</h3>
+            </div>
+        </div>
+        <footer class="footer-box">
+            <div class="footer-box__content">
+              <img class="text-is-centerme --modify4" src="/mf-logo2 white.svg">
```
Comment (Chrishalaby) [inline Home.html:143]: always add an alt to imgs

File: Home.css [hunk 7]
```diff
@@ -0,0 +1,294 @@
+body{
+    margin: 0;
+    padding: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+.navBar .navbar__image  {
+    max-width: 100px;
+    height: 100px;
```
Comment (Chrishalaby) [inline Home.css:8]: always use rem or em in measurement

File: Home.css [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+.navBar .navbar__image  {
+    max-width: 100px;
+    height: 100px;
+    float: left;
+    margin-left:10%;
+    padding: 0;
+}
+h1,h2{
```
Comment (Chrishalaby) [inline Home.css:13]: dont use html tags in css

File: Home.css [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+.icons a{
+    padding: 20px;
+    border-radius: 50px;
+}
+.--bottom p{
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    color: #99c3ff;
+}
+@media only screen and (min-width: 375px) and (max-width: 812px) {
+    /*.big__box{
```
Comment (Chrishalaby) [inline Home.css:279]: remove comments

**Review by Chrishalaby — CHANGES_REQUESTED** [review]
the BEM methodology was well worked on though it still needs a bit of improvements.

File: Home.html [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+                    </div>
+                </div>
+                <div class="__content --bottom">
+                    <p>Handcrafted by me &copy twentytwentytwo </p>
+                </div>
+            </div>
+            
+        </footer>
+    </body>
+</html>
```
Comment (Khousheish) [inline Home.html:160]: add EOF

**Review by Khousheish — CHANGES_REQUESTED** [review]
good progress in general just some issues that are repeating a lot please fix them in all places as I didnt comment on every violation
also fix both files to use 2 space indentation and pay close attention to the indentation since as mentioned lots of places had added spaces and no spaces etc..

File: Style.css [hunk 11]
```diff
@@ -0,0 +1,294 @@
...
+  --navyblue:rgb(14,14,51);
+}
+.navbar .navbar__image  {
+  max-width: 100px;
+  height: 100px;
+  float: left;
+  margin-left:10%;
+  padding: 0;
+}
+h1,
```
Comment (Chrishalaby) [inline Style.css:22]: dont use html tags in css, applied to everywhere in ur css code

File: Style.css [hunk 12]
```diff
@@ -0,0 +1,294 @@
...
+.navbar ul{
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  height: 100px;
+  text-align: right;
+}
+.navbar .navbar__text li{
+  margin-top: 30px;
```
Comment (Chrishalaby) [inline Style.css:36]: change all px in spacing margin padding etc.. to rem or em

File: Style.css [hunk 13]
```diff
@@ -0,0 +1,294 @@
...
+}
+.my-work{
+  display: flex;
+}
+.my-recent-work-img{
+  flex: 33.33%;
+  padding: 10px;
+}
+.above-footer{
+  position:absolute;
```
Comment (Chrishalaby) [inline Style.css:151]: it is preferred not to use position absolute

File: home.html [hunk 14]
```diff
@@ -0,0 +1,170 @@
...
+    <link rel="icon" type="image/x-icon" href="/mf-logo2 white.svg"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="Style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Russo+One&display=swap" rel="stylesheet">
+  </head>
+    <body>
+      <div class="navbar">
+        <ul>
+          <div class="navbar navbar__image">
```
Comment (Chrishalaby) [inline home.html:20]: div tag not needed here

File: home.html [hunk 15]
```diff
@@ -0,0 +1,170 @@
...
+             <img src= "/mf-logo2.svg" alt="logo">
+          </div>
+          <div class="navbar navbar__text">
```
Comment (Chrishalaby) [inline home.html:23]: same here no need for div, when u can apply directly the changes to the inner tags

File: home.html [hunk 16]
```diff
@@ -0,0 +1,170 @@
...
+            <li class="navbar navbar--text1"><a href="#">Mentorship</a> </li>
+            <li class="navbar navbar--text2"><a href="#">Say Hello</a> </li>
+             </div>
+        </ul>
+      </div>
+
+      <Section class="text-is-center">
+        <div class="text-is-center container">
```
Comment (Chrishalaby) [inline home.html:31]: both divs here should be removed and same idea should be applied everywhere else

File: home.html [hunk 17]
```diff
@@ -0,0 +1,170 @@
...
+            </a>
+            <a class="twit" href="https://linkedin.com/in/alin-torossian-225842232" role="button" target="_blank">
+              <i class="fa fa-linkedin"></i>
+            </a>
+            <a class="git" href="https://github.com/ALynn9" role="button" target="_blank">
+              <i class="fa fa-github"></i>
+            </a>
+          </div>
+        </div>
+        <div class="__content --bottom">
```
Comment (Chrishalaby) [inline home.html:163]: work on the BEM style

**Review by Chrishalaby — CHANGES_REQUESTED** [review]

File: Style.css [hunk 18]
```diff
@@ -0,0 +1,281 @@
...
+  margin-right: auto;
+  display: block;
+  width: 90%;
+  justify-content: center;
+}
+.my-recent-work{
+  padding-top: 2em;
+  text-align: center;
+}
+.my-recent-work img {
```
Comment (Chrishalaby) [inline Style.css:139]: remove all html tags from css
refer to them just by their class names

File: home.html [hunk 19]
```diff
@@ -0,0 +1,141 @@
...
+  <body>
+    <nav class="navbar">
+      <img class="navbar__image" src="/mf-logo2.svg" alt="logo">
+      <ul class="navbar__text">
+        <li> <a href="#" class="text--first">Mentorship</a> </li>
+        <li> <a href="#" class="text--second">Say Hello</a> </li>
+      </ul>
+    </nav>
+
+    <div class= "text-is-center title-sub">
```
Comment (Chrishalaby) [inline home.html:26]: add these 2 divs into a section maybe and so there wouldnt be any use for the div tags here

File: home.html [hunk 20]
```diff
@@ -0,0 +1,141 @@
...
+      <h1>Designer,Front-end developer & Student</h1>
+      <p>I design and code beautifully simple things, and i love what i do </p>
+      <img class="text-is-centerme" src="/my logo.svg" alt="logo">
+    </div>
+    <div class="container">
+      <img class="comp" src="hero.svg" alt="me-logo">
+    </div>
+
+    <section class="About-me">
+      <div class="title-sub --modify">
```
Comment (Chrishalaby) [inline home.html:36]: div tag can be removed here, changes can be applied to the section or the h2 ...

File: home.html [hunk 21]
```diff
@@ -0,0 +1,141 @@
...
+        quietly confident, naturally curious, and perpetually working on improving my chops one design problem at a time.</p>
+      </div>
+    </section>
+
+    <ul class="big__box">
+      <li class="big__box--boxFirst">
+        <img class="text-is-centerme --modify1" src="/designer.svg" alt="designer">
+        <h3>Designer</h3>
+        <p>I value simple content structure, clean design patterns, and thoughtful interactions.</p>
+        <div class="box__text--modify2">
```
Comment (Chrishalaby) [inline home.html:49]: why not directly apply the box text modify to the p tag instead of using a div

File: home.html [hunk 22]
```diff
@@ -0,0 +1,141 @@
...
+      <div class= my-recent-work__modify3">
+        <p>Here are a few design projects I've worked on recently. Want to see more?
+          <a href="#">Email me</a>
+        </p>
+      </div>
+      <figure class="my-work">
+        <div class="my-recent-work-img">
+          <img src="/wfdesignbuild.png" alt="work1">
+        </div>
+        <div class="my-recent-work-img">
```
Comment (Chrishalaby) [inline home.html:85]: a lot of un needed div tags

File: home.html [hunk 23]
```diff
@@ -0,0 +1,141 @@
...
+          <img src="/coupal.png" alt="work5">
+        </div>
+        <div class="my-recent-work-img">
+          <img src="/wedlerengineering.png"alt="work6">
+        </div>
+      </figure>
+    </section>
+
+    <div class="above-footer">
```
Comment (Chrishalaby) [inline home.html:105]: use sections

**Review by Chrishalaby — CHANGES_REQUESTED** [review]
A lot of div tags that can be removed, recheck your whole code and work on diminishing them

File: Home.html (hunk 1 above)
Comment (Alynn9) [reply Home.html]: I used the <nav> tag in navigation bar and removed unneeded div

File: Home.html (hunk 2 above)
Comment (Alynn9) [reply Home.html]: Div is removed

File: Home.html (hunk 4 above)
Comment (Alynn9) [reply Home.html]: I changed the h1 tag with h3 tag and i used h1 tag only once

File: Home.html (hunk 5 above)
Comment (Alynn9) [reply Home.html]: I substituted the divs witg ul tags

File: Home.html (hunk 6 above)
Comment (Alynn9) [reply Home.html]: I added the alt for all the images

File: Home.css (hunk 7 above)
Comment (Alynn9) [reply Home.css]: I changed all the margins and padding values into rem

File: Home.css (hunk 8 above)
Comment (Alynn9) [reply Home.css]: I removed all the html tags from css

File: Home.css (hunk 9 above)
Comment (Alynn9) [reply Home.css]: I fixed this and the website is responsive

File: Home.css [hunk 24]
```diff
@@ -0,0 +1,294 @@
...
+}
+.footer-box__content .text-is-centerm{
+    position: absolute;
+   
+}
+.--modify4 {
+    width: 100px;
+}
+.footer-box__content p{
+    color: #cce0fa;
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I created variables for the colors

File: Home.css [hunk 25]
```diff
@@ -0,0 +1,294 @@
+body{
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I changed home.css to Style.css

File: Home.css [hunk 26]
```diff
@@ -0,0 +1,294 @@
+body{
+    margin: 0;
+    padding: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+.navBar .navbar__image  {
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I changed to lowercase.

File: Home.css [hunk 27]
```diff
@@ -0,0 +1,294 @@
...
+    max-width: 100px;
+    height: 100px;
+    float: left;
+    margin-left:10%;
+    padding: 0;
+}
+h1,h2{
+    text-align: center;
+}
+.navBar  ul{
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I fixed this problem

File: Home.css [hunk 28]
```diff
@@ -0,0 +1,294 @@
...
+    height: 100px;
+    text-align: right;
+}
+.navBar .navbar__text li{
+    margin-top: 30px;
+    display: inline-block;
+}
+.navBar .navbar__text a {
+    text-decoration: none;
+    color:black;
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I changed all the color names into variables

File: Home.css [hunk 29]
```diff
@@ -0,0 +1,294 @@
...
+    width: 100px;
+    display: block;
+    padding: 10px 10px ;
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I fixed this problem when i used Ubuntu

File: Home.css [hunk 30]
```diff
@@ -0,0 +1,294 @@
...
+    position: relative;
+    border-radius: 12px;
+    width: 300px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 5%;
+}
+.img1  {
+    position: absolute;
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I used position absolute only once in above-footer class for the design

File: Home.css [hunk 31]
```diff
@@ -0,0 +1,294 @@
...
+    height: 400px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    border:1px solid rgb(223, 213, 213);
+    position:absolute;
+
+}
+.box3{
+    margin-left: 53.75%;
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I used display:flex and the problem solved

File: Home.css [hunk 32]
```diff
@@ -0,0 +1,294 @@
...
+    color: #cce0fa;
+    text-align: center;
+    font-size: x-large;
+    font-family: 'Russo One', sans-serif;
+}
+.--text3{
+    width: 400px;
+}
+
+#twit{
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I changed the ids into classes

File: Home.css [hunk 33]
```diff
@@ -0,0 +1,294 @@
...
+}
+.icons a{
+    padding: 20px;
+    border-radius: 50px;
+}
+.--bottom p{
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    color: #99c3ff;
+}
+@media only screen and (min-width: 375px) and (max-width: 812px) {
```
[the remark this answers is not in the document]
Comment (Alynn9) [reply Home.css]: I used @media (max-width :768px)

File: Home.html (hunk 10 above)
Comment (Alynn9) [reply Home.html]: The EOF is added

File: Style.css (hunk 11 above)
Comment (Alynn9) [reply Style.css]: I removed all the html tags from css

File: Style.css (hunk 12 above)
Comment (Alynn9) [reply Style.css]: I changed all the margins and padding values to rem instead of px

File: Style.css (hunk 13 above)
Comment (Alynn9) [reply Style.css]: Here the only place i used position:absolute for the design

File: home.html (hunk 14 above)
Comment (Alynn9) [reply home.html]: The div tags are removed

File: home.html (hunk 15 above)
Comment (Alynn9) [reply home.html]: I removed the div tag and the class directly to image

File: home.html (hunk 16 above)
Comment (Alynn9) [reply home.html]: I removed the div tags

File: home.html (hunk 17 above)
Comment (Alynn9) [reply home.html]: I tried my best to fix the BEM style

File: Style.css (hunk 18 above)
Comment (Alynn9) [reply Style.css]: I rechecked and removed the div tags instead i applied them by there class name

File: home.html (hunk 20 above)
Comment (Alynn9) [reply home.html]: I made the changes  and applied and used by class name

File: home.html (hunk 21 above)
Comment (Alynn9) [reply home.html]: I fixed it

File: home.html (hunk 22 above)
Comment (Alynn9) [reply home.html]: I removed all the unneeded tags

File: Style.css [hunk 34]
```diff
@@ -0,0 +1,343 @@
...
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+    margin-right: 5rem;
+  } 
+
+  .footer-box__content--logo {
+    margin-top: 20rem;
+  }
+}
```
Comment (Chrishalaby) [inline Style.css:343]: add extra blank line

File: Style.css [hunk 35]
```diff
@@ -0,0 +1,275 @@
+*{
+  margin: 0;
```
[the remark this answers is not in the document]
Comment (Khousheish) [reply Style.css]: no actually its not 0 on the elements
this article says it best 
https://css-tricks.com/reset-all-margins-padding/

File: Style.css [hunk 36]
```diff
@@ -0,0 +1,357 @@
+* {
+  box-sizing: border-box;
```
Comment (Khousheish) [inline Style.css:2]: readd the 0 margin and padding

File: Style.css [hunk 37]
```diff
@@ -0,0 +1,357 @@
...
+  --insta: #6559ca;
+  --lightgray: rgb(214,213,213);
+  --darkbrown: rgba(10,10,10,0.9);
+  --navyblue: rgb(14,14,51);
+  --black: rgb(0,0,0);
+  --white: rgb(255,255,255);
+  --purple:rgb(124,0,124);
+  --blue: rgb(8, 33, 255);
+  --fs-large: 1.75rem;
+  --fs-medium:1.25rem;
```
Comment (Khousheish) [inline Style.css:19]: add space

File: Style.css [hunk 38]
```diff
@@ -0,0 +1,357 @@
...
+.navbar__text {
+  display: flex;
+  list-style: none;
+  align-items: center;
+}
+
+.text--first {
+  text-decoration: none;
+  color: var(--black);
+  font-size:var(--fs-medium);
```
Comment (Khousheish) [inline Style.css:40]: add space

File: Style.css [hunk 39]
```diff
@@ -0,0 +1,357 @@
...
+  flex-direction: column;
+  margin: 0 auto;
+  padding: 1rem;
+  max-height: 300px;
+  max-width: 300px;
+}
+
+.title-sub__image--setup {
+  display: flex;
+  flex-direction: row;
```
Comment (Khousheish) [inline Style.css:76]: you can remove this since its row by default

File: Style.css [hunk 40]
```diff
@@ -0,0 +1,357 @@
...
+  max-height: 40rem;
+  color: white;
+  text-align: center;
+  font-size: var(--fs-large);
+}
+
+.about-me_modify {
+  font-size: var(--fs-medium);
+  margin: 0 2rem;
+  padding-top:2rem;
```
Comment (Khousheish) [inline Style.css:94]: add space

File: Style.css [hunk 41]
```diff
@@ -0,0 +1,357 @@
...
+  width: 60rem;
+  margin: 10rem auto auto auto;
+  border-radius: 1.25rem;
+  border: 2px solid var(--lightgray);
+  height: 25rem;
+  background-color: var(--white);
+  color: var(--black);
+  justify-content: center;
+  text-align: center;
+
```
Comment (Khousheish) [inline Style.css:109]: remove space

File: Style.css [hunk 42]
```diff
@@ -0,0 +1,357 @@
...
+}
+
+.big__box--boxes {
+  list-style: none;
+  text-decoration: none;
+  padding: 1rem 1rem;
+  font-size: var(--fs-medium);
+}
+
+.big__box--boxSecond {
```
Comment (Khousheish) [inline Style.css:126]: dont use camelcase in  class names

File: Style.css [hunk 43]
```diff
@@ -0,0 +1,357 @@
...
+  border-right: 2px solid var(--lightgray);
+  border-left: 2px solid var(--lightgray);
+  padding: 1rem 1rem;
```
Comment (Khousheish) [inline Style.css:129]: can be simplified to only 1 rem

File: Style.css [hunk 44]
```diff
@@ -0,0 +1,357 @@
...
+  list-style: none;
+  text-decoration: none;
+  font-size: var(--fs-medium);
+  width: auto;
```
Comment (Khousheish) [inline Style.css:133]: dont define auto width let it take as much as it needs

File: Style.css [hunk 45]
```diff
@@ -0,0 +1,357 @@
...
+}
+
+.modify-one {
+  padding-top: 1rem;
+  max-width: 3rem;
+  max-height: 3rem;
+}
+
+.box__text--modify-two {
+  padding:10px;
```
Comment (Khousheish) [inline Style.css:143]: add space

Comment (Khousheish) [reply Style.css:143]: you can write this as 1 rem

File: Style.css [hunk 46]
```diff
@@ -0,0 +1,357 @@
...
+  color: var(--blue);
+}
+
+.my-recent-work {
+  color: var(--darkbrown);
+  font-size: var(--fs-medium);
+  margin-top: 10rem;
```
Comment (Khousheish) [inline Style.css:150]: youre not following the scale from my previous discussion here

File: Style.css [hunk 47]
```diff
@@ -0,0 +1,357 @@
...
+}
+
+.my-recent-work__modify-three{
+  text-decoration: none;
+  color: var(--pink);
+}
+
+.my-work {
+  margin: 0 auto;
```
Comment (Khousheish) [inline Style.css:159]: remove

File: Style.css [hunk 48]
```diff
@@ -0,0 +1,357 @@
...
+  width: 80rem;
```
Comment (Khousheish) [inline Style.css:160]: remove random value

File: Style.css [hunk 49]
```diff
@@ -0,0 +1,357 @@
...
+}
+
+.my-work {
+  display: flex;
+}
+
+.my-recent-work-img {
+  flex-direction: column;
+  padding: 0.5rem;
+  width:100%;
```
Comment (Khousheish) [inline Style.css:181]: add space

File: Style.css [hunk 50]
```diff
@@ -0,0 +1,357 @@
...
+  border-radius: 2rem;
+}
+
+.above-footer {
+  position:absolute; /* i was obliged to use position absoulte other wise the design will be affected , i also tried to use z-index and changed the mardin to negative value in class footer-box instead of position absolute but it didn't work; */
+  background-color:var(--navyblue);
+  color: white;
+  width: 60rem;
```
Comment (Khousheish) [inline Style.css:189]: remove random rem value

File: Style.css [hunk 51]
```diff
@@ -0,0 +1,357 @@
...
+  margin-right: 10rem;
```
Comment (Khousheish) [inline Style.css:190]: follow the scale discussion we had last time
check the rest of the PR for this violation

File: Style.css [hunk 52]
```diff
@@ -0,0 +1,357 @@
...
+.icons__git {
+  color: var(--navyblue);
+  text-decoration: none;
+  border: 5px solid;
+  padding: 1rem 1.25rem;
+  border-radius: 80px;
+}
+
+.footer-box__icons {
+  margin:2rem 0;
```
Comment (Khousheish) [inline Style.css:278]: add space
please fix for all occurrences

File: Style.css [hunk 53]
```diff
@@ -0,0 +1,357 @@
...
+  font-size: var(--fs-medium);
+  display: flex;
+  column-gap: 1rem;
+  justify-content: center;
+}
+
+@media (max-width: 768px){
+  :root {
+    --fs-large: 2.25rem;
```
Comment (Khousheish) [inline Style.css:287]: you shouldnt change the scale if the screen gets smaller as itll automatically get smaller as youre using rems

File: Style.css [hunk 54]
```diff
@@ -0,0 +1,357 @@
...
+    --fs-medium:1.5rem;
+  }
+  
+  .about-me{
+    margin: 0;
```
Comment (Khousheish) [inline Style.css:292]: remove all these and readd margin and padding 0 in the *

**Review by Khousheish — CHANGES_REQUESTED** [review]
the first half of the css file is very good but towards the end I felt you didnt apply what we discussed before
please recheck my comments and apply them to the whole file as I didnt comment on every single occurrence

**Review by Khousheish — APPROVED** [review]
some comments dont seem t o be fully addressed please recheck them and resolve the ones that are done
other than that I think its good enough to move on to the next task
@Chrishalaby please recheck from your side

File: home.html [hunk 55]
```diff
@@ -0,0 +1,181 @@
...
+    <section class="about-me">
+      <h2>Hi,I'm Alynn.Nice to meet you</h2>
+      
+      <p class="about-me_modify">
+        Since beginning my journey as a freelance designer nearly 10 years ago, I've done remote work for agencies, consulted
+        for startups, and collaborated with talented people to create digital products for both business and consumer use. 
+        I'm quietly confident, naturally curious, and perpetually working on improving my chops one design problem at a time.
+      </p>
+      
+      <section class="big__box">
```
Comment (Chrishalaby) [inline home.html:53]: use a div here, never use a section inside another section

File: home.html [hunk 56]
```diff
@@ -0,0 +1,181 @@
...
+    <footer class="footer-box">
+        <img 
+          class="footer-box__content--logo" 
+          src="./mf-logo2 white.svg" 
+          alt="logo-white"
+        >
+    
+        <p class="footer-box--text">Living,Learning & leveling up one day at a time.</p>
+       
+        <div class="footer-box__icons">
```
Comment (Chrishalaby) [inline home.html:149]: is the div here needed?

File: home.html [hunk 57]
```diff
@@ -0,0 +1,181 @@
...
+        </li>
+        
+        <li class="li-tag"> 
+          <img class="my-recent-work-img" src="./wedlerengineering.png"alt="work6">
+        </li>
+      </ul>
+    </section>
+    
+    <section class="above-footer">
+      <div class="above-footer--container">
```
Comment (Chrishalaby) [inline home.html:131]: can div here be removed?

**Review by Chrishalaby — APPROVED** [review]

**Pull request merged**